src/App.js

```diff
@@ -0,0 +1,82 @@
+import "macro-css";
+import './index.scss';
+import { MainTemperatureBlock } from "./components/DateTemperature/MainTemperatureBlock/index.js";
+import { SearchToolbar } from "./components/SearchToolbar";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+import { DayTemperatureBlock } from "./components/DayTemperature";
+
+
+const indexOfItem = [2, 5, 8, 11, 14, 17, 20, 23]
+function App() {
+  const getTemperature = (temperature) => temperature > 0 ? "+" + Math.floor(temperature) + "°" : Math.floor(temperature) + "°";
+  const [data, setData] = useState({
+    city: "Vinnytsya",
+    country: "",
+    list: [],
+    days: [],
+    current: {
+      icon: "",
+      temp: "",
+      text: "",
+    }
+  });
+  // console.log(data.days);
+  const [time, setTime] = useState({
```

Comment:
no need object. u can just use the new Date() with the correct format. if u no need time here why do u pass it as props? use it inside needed component